### DISCUSSION TITLE
Using Pyfluent in Spyder

### DISCUSSION THREAD

**Discussion by ToMdAvI** [ansys/pyfluent#3130] [Q&A]
Hello,

I have been using PyFluent in Visual Studio code for a while now, but would like to use it within Spyder.

I have downloaded Spyder and ran the same code as from VS, but it returns this error in photo.

![Spyder_Error](https://github.com/user-attachments/assets/27b5e6cd-4834-4c52-8626-7f51e779520f)

How do I install the ansys module in Spyder or give Spyder access to the paths where i have the packages installed?

- Anaconda/Spyder are downloaded on my laptop, like VS
- Ansys Packages are downloaded on a hard drive

Thanks

**Comment by Jojo2587** [comment]
You need to install PyFluent as well. In the command line:  **pip install pyfluent**
